File: Max Consecutive Ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max=-1
        temp=0
        for i in nums:
            if i == 0:
                if(max<temp):
                    max=temp
                temp=0
            else:
                temp+=1
        if(max<temp):
            max=temp
        return max

File: Leetcode/Arrays 101/Introduction/MaxConsecutiveOnes.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int max =-1, temp=0;
        for (auto it = nums.begin(); it != nums.end(); it++) {
            if(*it == 0){
                if(temp>max)
                    max=temp;
                temp=0;    
            }
            else{
                temp++;
                }
        }
        if(temp>max){
            max=temp;
        }
        return max;
    }
};
